.github/workflows/codeql.yml

```
@@ -45,11 +45,17 @@ jobs:
       run: |
         echo "CHECKOUT_REF=$(git symbolic-ref HEAD)" >> "$GITHUB_ENV"
         echo "CHECKOUT_SHA=$(git rev-parse HEAD)" >> "$GITHUB_ENV"
+        echo "CHECKOUT_BRANCH_NAME=$(git rev-parse --abbrev-ref HEAD) >> "$GITHUB_ENV"
+
+    - name: Set experimental environment variable for 8.x branch
+        if: env.CHECKOUT_BRANCH_NAME == '8.x'
```

I guess we can change it to
```suggestion
        if: env. CHECKOUT_REF == 'refs/heads/8.x'
```